packages/cli/test/unit/util/deploy/create-git-meta.test.ts

```diff
@@ -0,0 +1,166 @@
+import { join } from 'path';
+import fs from 'fs-extra';
+import { getWriteableDirectory } from '@vercel/build-utils';
+import {
+  createGitMeta,
+  getRepoData,
+  parseRepoUrl,
+} from '../../../../src/util/deploy/create-git-meta';
+import { client } from '../../../mocks/client';
+
+const fixture = (name: string) =>
+  join(__dirname, '../../../fixtures/unit/create-git-meta', name);
+
+describe('getRepoData', () => {
+  it('gets repo data for no-email', async () => {
+    const configPath = join(fixture('no-email'), 'git/config');
+    const data = await getRepoData(configPath, client.output);
+    expect(data?.provider).toEqual('github');
+    expect(data?.org).toEqual('MatthewStanciu');
+    expect(data?.repo).toEqual('git-test');
+  });
+  it('gets repo data for no-origin', async () => {
+    const configPath = join(fixture('no-origin'), 'git/config');
+    const data = await getRepoData(configPath, client.output);
+    expect(data).toBeUndefined();
+  });
+  it('displays debug message when repo data cannot be parsed', async () => {
+    const dir = await getWriteableDirectory();
+    client.output.debugEnabled = true;
+    const data = await getRepoData(join(dir, 'git/config'), client.output);
+    expect(data).toBeUndefined();
+    expect(
+      client.outputBuffer.includes('Error while parsing repo data'),
+      'Debug message was not found'
+    ).toBeTruthy();
+  });
+});
+
+describe('parseRepoUrl', () => {
+  it('should parse github https url', () => {
+    const parsedUrl = parseRepoUrl('https://github.com/vercel/vercel.git');
```

Comment:
Lets add another test for `parseRepoUrl('https://github.com/vercel/vercel')` since I know that github supports this without the `.git` suffix

Reply:
Oh also lets test `vercel/next.js` just to be sure org and repo and assign properly 🙂